tests/x11/yast2_snapper.pm

```diff
@@ -46,7 +46,7 @@ sub clean_and_quit() {
     # C'l'ose  the snapper module
     wait_screen_change { send_key "alt-l" };
     # Wait until xterm is focussed, delete the directory and close xterm
-    wait_idle;
+    assert_screen 'xterm',        10;
```

Comment:
never use `assert_screen` with less than default timeout unless you are waiting for something which is time-critical itself, e.g. "reboot now" dialog or grub menue with timeout -> just use `assert_screen 'xterm';`